### PR TITLE
Api prep and autoloading

### PR DESCRIPTION
**Pull request by billythakidd04**
- set up composer to do autoloading across the entire project (within the scope of week 9 directory)
- cleaned up a bunch of formatting
- renamed some functions to better self document
- fixed namespacing
- added htaccess file for slim
- required slim in composer
- fix a few bugs around the student and group classes

To test this open the week 9 path and ensure the students and groups are showing correctly and there are no errors popping up.
This does **NOT** implement the actual api, just clean up and prep for it. Also some housekeeping I've been meaning to do.

Resources:
https://code.tutsplus.com/tutorials/how-to-autoload-classes-with-composer-in-php--cms-35649
https://www.slimframework.com/docs/v4/start/installation.html
https://getcomposer.org/doc/01-basic-usage.md#autoloading
https://www.slimframework.com/docs/v4/start/web-servers.html

this needs to be merged **BEFORE** #44

### CODE REVIEW

**Review by margeschrec — DISMISSED** [review]

File: Week_9_PHP_APIs/composer.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (nburd90) [inline Week_9_PHP_APIs/composer.json:1]: Should we be doing something similar as this to our composer.json file?

**Review by rbaia28 — COMMENTED** [review]
Im getting an error 404 when im on the branch trying to open in my browser

**Comment by eiman-kased** [comment]
> Im getting an error 404 when im on the branch trying to open in my browser

Same!

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (nburd90) [inline composer.json]: Same question here.

**Comment by billythakidd04** [comment]
@eiman-kased, @rbaia28 @margeschrec and @nburd90 we will cover this whole set of changes in class today.

**Review by rbaia28 — APPROVED** [review]

**Review by vricci518 — APPROVED** [review]
I get an err

**Review by margeschrec — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by eiman-kased — APPROVED** [review]

**Review by nburd90 — APPROVED** [review]

**Review by gc824 — APPROVED** [review]

**Pull request merged**